### PR TITLE
NXP-24337: Add rvt, dlx to mimetype mappings

### PR DESCRIPTION
**Pull request by covolution**
t&p failed on an integration test (due to missing commit) but I am running it again.

### CODE REVIEW

**Comment by efge** [comment]
👍

**Comment by ataillefer** [comment]
:+1:

File: nuxeo-core/nuxeo-core-mimetype/src/test/java/org/nuxeo/ecm/platform/mimetype/TestMimetypeSniffing.java [hunk 1]
```diff
@@ -122,6 +122,12 @@ public void testVisioDocument() {
                 mimetypeRegistry.getMimetypeFromFile(getFileFromResource("test-data/hello.vsd")));
     }
 
+    @Test
+    public void testCustomMimetypExtensions() {
```
Comment (jcarsique) [inline nuxeo-core/nuxeo-core-mimetype/src/test/java/org/nuxeo/ecm/platform/mimetype/TestMimetypeSniffing.java]: typo in method name

**Comment by covolution** [comment]
Not needed at the moment

**Pull request closed without merging**